src/lightning_app/core/app.py

```diff
@@ -45,6 +46,7 @@ class LightningApp:
     def __init__(
         self,
         root: "lightning_app.LightningFlow",
+        flow_compute_config: Optional["lightning_app.CloudCompute"] = None,
```

Comment:
This is great. Two minor comments:

- We call the class "CloudCompute", so I would avoid CloudConfig or config in the name of arguments, or in the docstring explanation. 
- The argument in work is called simply "cloud_compute". For the app here, I propose "flow_cloud_compute" or "cloud_compute" as the input argument names.

Reply:
agreed. renamed to `flow_cloud_compute`.